docs/source/modules/tutorial.rst

```diff
@@ -0,0 +1,628 @@
+4. ``model-tools``: helper functions that translate from machine learning models
+   to brain models to be tested on brain-score.
+5. ``brainio_collection``: the repo that packages and collects the stimuli/data.
+6. ``brainio_base``: repo that contains various data structures for ``BrainIO``.
+7. ``result_caching``: a helper repo to store the results of function calls so they can
+   be re-used without re-computing.
+8. ``brain-score.web``: website front and back end.
+9. ``Brainio_contrib`` (archived): used in the past to contribute stimuli and datasets
+   (now part of ``brainio_collection``).
+
+Which repo(s) will I use?
+-----------------------
+When we get to the install guide, we will show you exactly how to
+clone/fork a repo for your own project in the easiest way possible.
+But for now, you will mainly only need the ``sample-model-submission`` repo.
+
+How do I get a Brain-Score for my model?
+----------------------------------------
+When you submit a model to our website, it is scored against all
+availible benchmarks (e.g. neural predictivity on IT recordings
+from Majaj*, Hong* et al. 2015; see benchmarks_ for more details). The (hierarchical) mean of
+individual benchmark scores is the Brain-Score itself.
+
+Before submitting your model, you might want to get a quick sense of its performance;
+to that end, we provide *public* benchmarks that you can run locally, which are different subsets
+of the larger benchmark dataset. This is mainly used to optimize your model before
+submission, or if you want to score models locally on publicly available data.
+*Note: a submission is the only way to score models on private evaluation data.*
+
+
+
+
+
+Why do you recommend installing and submitting the way outlined in this guide? In other words, why should I do it your way?
+------------------------------------------------------------------------------
+
+A reasonable question, and it is always good to be skeptical. The short answer
+is that using an IDE like Pycharm or VSCode along with virtual environments
+drastically cuts the error rate for install down, as well as makes the whole
+process of installing dependencies easier. Using a venv also helps with headaches
+caused by clashes between Anaconda and PIP, and Pycharm
+(or another IDE like VSCode) takes care of that.
+
+Do I have to read/do this entire tutorial to submit a model?
+------------------------------------------------------------
+
+No - You can just read the ``Quickstart`` section, if you do not
+wish to read/do this entire tutorial. However, we recommend referring back to this
+tutorial to help with errors that might pop up along the way.
+
+
+
+
+Install Brain-Score Repos and Dependencies
+==========================================
+In this section, we will show you how to get packages installed and dependencies
+linked in order to run setup for submission and scoring.
+
+1. Download PyCharm (https://www.jetbrains.com/pycharm/download/#section=windows) or another IDE.
```

Comment:
just noticed this -- remove the `#section=windows` to make it agnostic of the user's OS